### PR TITLE
Update Bitrix24

### PR DESCRIPTION
**Pull request by rakleed**
- Add regional domains.

### CODE REVIEW

**Comment by Carlgo11** [comment]
With my limited knowledge about Bitrix24, it looks like they operate in Europe and the U.S. as [bitrix24.com](https://bitrix24.com)/[bitrix24.eu](https://bitrix24.eu) (Bitrix24 Ltd. / Bitrix, Inc.) whilst operating in Russia with [bitrix24.ru](https://bitrix24.ru) (1C-Bitrix OOO).

If you want to list the Russian site I suggest you create a different entry for bitrix24.ru and specify `ru` in the regions array. At the same time you can specify `-ru` in the regions array for bitrix24.com.

As a site note, what's the difference between Б and в? 🤨

**Comment by rakleed** [comment]
Done.

Б/б is pronounced like B/b, and В/в is like V/v. Well, they are different letters. Why are you asking?

File: entries/b/bitrix24.ru.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "Битрикс24>": {
+    "domain": "bitrix24.ru",
+    "additional-domains": [
+      "bitrix24.net"
+    ],
```
Comment (phallobst) [inline entries/b/bitrix24.ru.json]: You can just re-use the original icon here and remove `img/b/bitrix24.ru.svg`.
```suggestion
    ],
    "img": "bitrix24.com.svg",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/b/bitrix24.ru.json (hunk 1 above)
Comment (phallobst) [inline entries/b/bitrix24.ru.json]: The `additional-domains` are primarily targeted at password managers. I'm not sure how they are able to properly handle duplicates across different entries.

Comment (rakleed) [reply entries/b/bitrix24.ru.json]: Done.

Comment (rakleed) [reply entries/b/bitrix24.ru.json]: I can remove an additional domain, but usually saving the login and password is attached to the login page.

**Comment by phallobst** [comment]
I'm actually not quite sure why we need a separate entry for the `.ru` domain in the first place. It would probably be better to just list all the `additional-domains` in the original one, especially since they seem to share a common login page.

**Comment by rakleed** [comment]
@Carlgo11 I have added additional regional domains as suggested by @phallobst since I have checked in the media and on [Wikipedia[ru]](https://ru.wikipedia.org/wiki/1С-Битрикс_(компания)) and nowhere is it mentioned that they are different companies. In general, in 2007, Bitrix created a joint venture with 1C, and therefore their company is called 1C-Bitrix. And for the European and American markets, they have registered separate companies, most likely to simplify the interaction with their customers.

**Review by phallobst — APPROVED** [review]

**Pull request merged**